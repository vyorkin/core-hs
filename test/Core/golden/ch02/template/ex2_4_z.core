W x y = x y y

main = W S K 42
