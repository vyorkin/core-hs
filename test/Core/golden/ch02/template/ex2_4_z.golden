2    : NSupercomb main

14   : NAp 12 13

12   : NAp 11 4
14   : NAp 12 13

11   : NAp 1 6
12   : NAp 11 4
14   : NAp 12 13

1    : NSupercomb W
11   : NAp 1 6
12   : NAp 11 4
14   : NAp 12 13

16   : NAp 15 4
14   : NAp 12 13

15   : NAp 6 4
16   : NAp 15 4
14   : NAp 12 13

6    : NSupercomb S
15   : NAp 6 4
16   : NAp 15 4
14   : NAp 12 13

19   : NAp 17 18

17   : NAp 4 13
19   : NAp 17 18

4    : NSupercomb K
17   : NAp 4 13
19   : NAp 17 18

13   : NNum 42

Stats:
  Total number of steps: 11